src/Faker/Combinators.hs

```diff
@@ -113,3 +113,7 @@ frequency xs0 = fromRange (1, tot) >>= (`pick` xs0)
 --
 fakeEnumFromTo :: Enum a => a -> a -> Fake a
 fakeEnumFromTo from to = toEnum <$> fromRange (fromEnum from, fromEnum to)
+
+-- | A sumtype can just derive these `fakeEnumFromTo` `minBound` `maxBound`
+fakeBoundedEnum :: Enum a => Bounded a => Fake a
```

Comment:
I think this type signature would be much clearer: `(Enum a, Bounded a) => Fake a`

Also, can you add a haddock since annotation and update the changelog. Thanks!